packages/shared/src/TypedMessage/core.ts

```diff
@@ -103,7 +103,7 @@ export function makeTypedMessageUnknown(raw?: unknown, meta?: Meta): TypedMessag
 export interface TypedMessageEmpty extends SerializableTypedMessage<1> {
     readonly type: 'empty'
 }
-export const isTypedMessageEmpty = createIsType<TypedMessageEmpty>('unknown')
+export const isTypedMessageEmpty = createIsType<TypedMessageEmpty>('empty')
```

Comment:
I thought this was a mistake and fixed it. Let me know if it was intentional

Reply:
yeah it's mistake